gst/datarepo/gstdatareposink.c

```diff
@@ -331,7 +331,7 @@ gst_data_repo_sink_write_flexible_tensors (GstDataRepoSink * sink,
     }
 
     if (!gst_tensor_meta_info_parse_header (&meta, info.data)) {
-      GST_ERROR_OBJECT (sink, "Invalid flexible tensors");
+      GST_ERROR_OBJECT (sink, "Invalid tensors, tensors format is static.");
```

Comment:
Please revise the msg. Invalid tensors' format? Invalid formats of tensors?

Reply:
Thank you. I will fix the msg.